TSC-Charter.md

```diff
@@ -84,25 +63,25 @@ including:
 * Conduct guidelines.
 * Maintaining the list of additional Collaborators.
 * Development process and coding standards.
-* Mediating technical conflicts between Collaborators or ASC Organization
+* Mediating technical conflicts between Collaborators or Planetary Software Organization
```

Comment:
Do we want to add 'Maintaining a list of affiliate projects'?

Reply:
Sure.